catkit/hardware/standa/stages.py

```diff
@@ -246,44 +268,48 @@ def get_position(self):
 
         :return: stagePosition Position of the stage
         """
-        return self.conversionFactor * self.get_enc_position()[1]
+        return self.conversionFactor * self.get_enc_position()
 
     def stop(self):
         result = self.instrument_lib.lib.command_sstp(self.instrument)
         if result != self.instrument_lib.Result.Ok:
             raise RuntimeError("Soft stop failed")
 
-    def await_stop(self, *args, **kwargs):
+    def await_stop(self,  timeout=10*60, poll_interval=1):
         """ Wait for device to indicate it has stopped moving.
 
             See catkit.util.poll_status for API.
+
+            NOTE: Default timeout is arbitrarily long.

```

Comment:
TODO: We could either make sure this is really long enough or calc an estimate based on speed.